server/src/main/java/org/elasticsearch/indices/IndicesService.java

```diff
@@ -1219,7 +1220,13 @@ public void run() {
             }
             // Reschedule itself to run again if not closed
             if (closed.get() == false) {
-                threadPool.schedule(this, interval, ThreadPool.Names.SAME);
+                try {
+                    threadPool.schedule(this, interval, ThreadPool.Names.SAME);
```

Comment:
easier to use `scheduleUnlessShuttingDown`?

Reply:
Oh sorry! Didn't know about that method, will open cleanup PR in a sec.